stable/external-dns/README.md

```diff
@@ -0,0 +1,56 @@
+# external-dns chart
+
+## Chart Details
+
+This chart will do the following:
+
+* Create a deployment of [external-dns] within your Kubernetes Cluster.
+
+Currently this uses the [Zalando] hosted container, if this is a concern follow the steps in the [external-dns] documentation to compile the binary and make a container. Where the chart pulls the image from is fully configurable.
+
+## Notes
+
+You probably want to make sure the nodes have IAM permissions to modify the R53 entries. More on this later.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/external-dns
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the external-dns chart and their default values.
+
+
+| Parameter              | Description                                                                                                      | Default                                                      |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| `domainFilter`         | Limit possible target zones by a domain suffix (optional).                                                       | `disabled`                                                   |
+| `extaArg`              | Optional object of extra args, as `name`: `value` pairs. Where the name is the command line arg to external-dns. | ``                                                           |
+| `image.name`           | Container image name (Including repository name if not `hub.docker.com`).                                        | `registry.opensource.zalan.do/teapot/external-dns`           |
+| `image.pullPolicy`     | Container pull policy.                                                                                           | `IfNotPresent`                                               |
+| `image.tag`            | Container image tag.                                                                                             | `v0.3.0`                                                     |
+| `podAnnotations`       | Additional annotations to apply to the pod.                                                                      | `None`                                                       |
+| `policy`               | Modify how DNS records are sychronized between sources and providers (options: sync, upsert-only ).              | `upsert-only`                                                |
+| `provider`             | The DNS provider where the DNS records will be created (options: aws, google, inmemory, azure ).                 | `aws`                                                        |
```

Comment:
Full list of supported providers as of `v0.4`: `aws, google, azure, cloudflare, digitalocean, inmemory`